packages/common/src/utils/poll.ts

```diff
@@ -11,14 +11,16 @@
  * @param validationFn
  * @param opts
  */
```

Comment:
❤️❤️❤️❤️ Only as is that the polling also implement an incremental back-off - so if it's not valid after 300ms, the next timeout is 600ms. This is what AGO asked us to do if we use a polling strategy b/c the api is not deterministic

Reply:
I thought about this too. Will push up a quick change to add incremental backoff